from PyQt4 import QtCore, QtGui

try:
    _fromUtf8 = QtCore.QString.fromUtf8
except AttributeError:
    def _fromUtf8(s):
        return s

try:
    _encoding = QtGui.QApplication.UnicodeUTF8
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig, _encoding)
except AttributeError:
    def _translate(context, text, disambig):
        return QtGui.QApplication.translate(context, text, disambig)

class Ui_Recorridos(QtGui.QMainWindow):
    def setupUi(self, Recorridos):
        Recorridos.setObjectName(_fromUtf8("Recorridos"))
        Recorridos.resize(640, 520)
        Recorridos.setStyleSheet(("background-image: url(imagenes/principal.jpg)"))
        self.timeEHoraS = QtGui.QTimeEdit(Recorridos)
        self.timeEHoraS.setGeometry(QtCore.QRect(140, 380, 151, 27))
        self.timeEHoraS.setObjectName(_fromUtf8("timeEHoraS"))
        self.timeEHoraLl = QtGui.QTimeEdit(Recorridos)
        self.timeEHoraLl.setGeometry(QtCore.QRect(350, 380, 161, 27))
        self.timeEHoraLl.setObjectName(_fromUtf8("timeEHoraLl"))
        self.lHoraSalida = QtGui.QLabel(Recorridos)
        self.lHoraSalida.setGeometry(QtCore.QRect(140, 350, 161, 20))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lHoraSalida.setFont(font)
        self.lHoraSalida.setObjectName(_fromUtf8("lHoraSalida"))
        self.lHoraLlegada = QtGui.QLabel(Recorridos)
        self.lHoraLlegada.setGeometry(QtCore.QRect(350, 350, 181, 17))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lHoraLlegada.setFont(font)
        self.lHoraLlegada.setObjectName(_fromUtf8("lHoraLlegada"))
        self.lIDConductor = QtGui.QLabel(Recorridos)
        self.lIDConductor.setGeometry(QtCore.QRect(70, 90, 151, 20))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lIDConductor.setFont(font)
        self.lIDConductor.setObjectName(_fromUtf8("lIDConductor"))
        self.lIDUnidad = QtGui.QLabel(Recorridos)
        self.lIDUnidad.setGeometry(QtCore.QRect(110, 130, 111, 20))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lIDUnidad.setFont(font)
        self.lIDUnidad.setObjectName(_fromUtf8("lIDUnidad"))
        self.lineEIDConductor = QtGui.QLineEdit(Recorridos)
        self.lineEIDConductor.setGeometry(QtCore.QRect(230, 90, 301, 27))
        self.lineEIDConductor.setObjectName(_fromUtf8("lineEIDConductor"))
        self.lineEIDUnidad = QtGui.QLineEdit(Recorridos)
        self.lineEIDUnidad.setGeometry(QtCore.QRect(230, 130, 301, 27))
        self.lineEIDUnidad.setObjectName(_fromUtf8("lineEIDUnidad"))
        self.lineEPasajeros = QtGui.QLineEdit(Recorridos)
        self.lineEPasajeros.setGeometry(QtCore.QRect(350, 180, 81, 27))
        self.lineEPasajeros.setObjectName(_fromUtf8("lineEPasajeros"))
        self.lPasajeros = QtGui.QLabel(Recorridos)
        self.lPasajeros.setGeometry(QtCore.QRect(230, 180, 111, 20))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lPasajeros.setFont(font)
        self.lPasajeros.setObjectName(_fromUtf8("lPasajeros"))
        self.comboBEstSalida = QtGui.QComboBox(Recorridos)
        self.comboBEstSalida.setGeometry(QtCore.QRect(290, 240, 251, 27))
        self.comboBEstSalida.setObjectName(_fromUtf8("comboBEstSalida"))
        self.comboBEstSalida.addItem(_fromUtf8(""))
        self.comboBEstSalida.addItem(_fromUtf8(""))
        self.comboBEstSalida.addItem(_fromUtf8(""))
        self.comboBEstSalida.addItem(_fromUtf8(""))
        self.comboBEstSalida.addItem(_fromUtf8(""))
        self.comboBEstSalida.addItem(_fromUtf8(""))
        self.lEstacionSalida = QtGui.QLabel(Recorridos)
        self.lEstacionSalida.setGeometry(QtCore.QRect(110, 240, 171, 20))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lEstacionSalida.setFont(font)
        self.lEstacionSalida.setObjectName(_fromUtf8("lEstacionSalida"))
        self.lEstacionLlegada = QtGui.QLabel(Recorridos)
        self.lEstacionLlegada.setGeometry(QtCore.QRect(90, 290, 191, 20))
        font = QtGui.QFont()
        font.setPointSize(14)
        font.setBold(True)
        font.setWeight(75)
        self.lEstacionLlegada.setFont(font)
        self.lEstacionLlegada.setObjectName(_fromUtf8("lEstacionLlegada"))
        self.comboBEstLlegada = QtGui.QComboBox(Recorridos)
        self.comboBEstLlegada.setGeometry(QtCore.QRect(290, 290, 251, 27))
        self.comboBEstLlegada.setObjectName(_fromUtf8("comboBEstLlegada"))
        self.comboBEstLlegada.addItem(_fromUtf8(""))
        self.comboBEstLlegada.addItem(_fromUtf8(""))
        self.comboBEstLlegada.addItem(_fromUtf8(""))
        self.comboBEstLlegada.addItem(_fromUtf8(""))
        self.comboBEstLlegada.addItem(_fromUtf8(""))
        self.comboBEstLlegada.addItem(_fromUtf8(""))
        self.lRecorridos = QtGui.QLabel(Recorridos)
        self.lRecorridos.setGeometry(QtCore.QRect(230, 40, 181, 20))
        font = QtGui.QFont()
        font.setPointSize(20)
        font.setBold(True)
        font.setItalic(True)
        font.setWeight(75)
        self.lRecorridos.setFont(font)
        self.lRecorridos.setObjectName(_fromUtf8("lRecorridos"))
        self.bingresarRecorridos = QtGui.QPushButton(Recorridos)
        self.bingresarRecorridos.setGeometry(QtCore.QRect(340, 440, 240, 50))
        self.bingresarRecorridos.setObjectName(_fromUtf8("bingresarRecorridos"))
        self.bRegresarRecorridos = QtGui.QPushButton(Recorridos)
        self.bRegresarRecorridos.setGeometry(QtCore.QRect(50, 440, 240, 50))
        self.bRegresarRecorridos.setObjectName(_fromUtf8("bRegresarRecorridos"))

        self.retranslateUi(Recorridos)
        QtCore.QMetaObject.connectSlotsByName(Recorridos)

    def retranslateUi(self, Recorridos):
        Recorridos.setWindowTitle(_translate("Recorridos", "Recorridos", None))
        self.lHoraSalida.setText(_translate("Recorridos", "HORA DE SALIDA", None))
        self.lHoraLlegada.setText(_translate("Recorridos", "HORA DE LLEGADA", None))
        self.lIDConductor.setText(_translate("Recorridos", "ID CONDUCTOR", None))
        self.lIDUnidad.setText(_translate("Recorridos", "ID UNIDAD", None))
        self.lPasajeros.setText(_translate("Recorridos", "PASAJEROS", None))
        self.comboBEstSalida.setItemText(0, _translate("Recorridos", "PISCINA OLIMPICA", None))
        self.comboBEstSalida.setItemText(1, _translate("Recorridos", "ESPOL", None))
        self.comboBEstSalida.setItemText(2, _translate("Recorridos", "ACACIAS", None))
        self.comboBEstSalida.setItemText(3, _translate("Recorridos", "TERMINAL", None))
        self.comboBEstSalida.setItemText(4, _translate("Recorridos", "ORQUIDEAS", None))
        self.comboBEstSalida.setItemText(5, _translate("Recorridos", "DURAN", None))
        self.lEstacionSalida.setText(_translate("Recorridos", "ESTACION SALIDA", None))
        self.lEstacionLlegada.setText(_translate("Recorridos", "ESTACION LLEGADA", None))
        self.comboBEstLlegada.setItemText(0, _translate("Recorridos", "PISCINA OLIMPICA", None))
        self.comboBEstLlegada.setItemText(1, _translate("Recorridos", "ESPOL", None))
        self.comboBEstLlegada.setItemText(2, _translate("Recorridos", "ACACIAS", None))
        self.comboBEstLlegada.setItemText(3, _translate("Recorridos", "TERMINAL", None))
        self.comboBEstLlegada.setItemText(4, _translate("Recorridos", "ORQUIDEAS", None))
        self.comboBEstLlegada.setItemText(5, _translate("Recorridos", "DURAN", None))
        self.lRecorridos.setText(_translate("Recorridos", "RECORRIDOS", None))

